Lib/fontTools/varLib/mutator.py

```diff
@@ -121,6 +122,46 @@ def instantiateVariableFont(varfont, location, inplace=False):
 		log.info("Building interpolated tables")
 		merger.instantiate()
 
+	addidef = False
+	for glyph in glyf.glyphs.values():
+		if addidef:
+			break
+		if hasattr(glyph, "program"):
+			instructions = glyph.program.getAssembly()
+			# If GETVARIATION opcode is used in bytecode of any glyph add IDEF
+			addidef = any(op.startswith("GETVARIATION") for op in instructions)
```

Comment:
You could break the loop here instead of at the next iteration.

Reply:
Maybe a bit ugly, but you can remove the addifdef completely and use a "for else" block.

Reply:
Hmm I don't think "for else" is applicable here, since we want to implement logic if breaking out of loop?  Also, I intended to have addidef block separately to make it obvious in the code that the idef is not always added.

Reply:
Right.